spdx-utils/src/test/kotlin/SpdxDeclaredLicenseMappingTest.kt

```diff
@@ -77,10 +77,13 @@ class SpdxDeclaredLicenseMappingTest : WordSpec({
 
         "be case-insensitve" {
             val map = SpdxDeclaredLicenseMapping.mapping
-            map.forEach { (key, license) ->
-                map[key.toLowerCase()] shouldBe license
-                map[key.toUpperCase()] shouldBe license
-                map[key.toLowerCase().capitalize()] shouldBe license
+
```

Comment:
consider applying the `forAll` approach here as well as I outlined for another commit in this PR.

Reply:
This still requires the `assertSoftly()` / `forEach()` appraoch as we need both key and value here and `forAll()` is not available for maps.

Reply:
Oh I see thanks. Haven't checked but maybe `asSequence().forAll {` is an option, totally up to you.

Reply:
Ah, that indeed works, done.